### ISSUE TITLE
About training details.

### ISSUE THREAD

**Issue by WY-2022** [ronferens/hyperpose#1]
I was wondering if you might be willing to share how you arrived at your training hyper-parameters—for instance, the batch size, number of epochs, etc.—as I wasn’t able to find those specifics in the paper. Additionally, did you reserve a separate validation set for tuning these parameters?

Any details you could offer would be greatly appreciated. Thank you!

**Comment by ronferens** [comment]
Thank you @WY-2022 for your interest in our work.
The configurations used for training both the single- and multi-scene models are detailed in the .JSON files under the [configs folder](https://github.com/ronferens/hyperpose/tree/main/configs). As we noted in the supplemetary material, we've followed a three-steps training procedure detailed in our paper [Do We Really Need Scene-specific Pose Encoders?
](https://www.computer.org/csdl/proceedings-article/icpr/2021/09412225/1tmjC49HiFi)
As you correcly suggested, we've used the common ”best model” selection practice, by selecting the optimal training hyperparameters (batch#, batch size, optimal epoch, learning-rate, etc.) using a validation set. We then evaluate the model on the test set.

**Comment by WY-2022** [comment]
Thank you for your answer. It seems that the supplementary materials of your paper have not been published by CVPR yet. I will continue to follow the progress of the conference and read the paper mentioned in your answer.
